x-pack/plugins/security_solution/server/connectors/sentinelone/sentinelone.ts

```diff
@@ -65,6 +66,8 @@ export class SentinelOneConnector extends SubActionConnector<
       agents: `${this.config.url}${API_PATH}/agents`,
     };
 
+    this.endpointAppContextService = params.endpointAppContextService;
```

Comment:
also when registering `.sentinelone` connector from Security Solution we would have access to the `.endpointAppContextService`